test/e2e/lifecycle/cluster_upgrade.go

```diff
@@ -446,9 +446,6 @@ func runUpgradeSuite(
 	upgradeType upgrades.UpgradeType,
 	upgradeFunc func(),
 ) {
-	upgCtx, err := getUpgradeContext(f.ClientSet.Discovery(), *upgradeTarget)
```

Comment:
getUpgradeContext does sanity checks (server is reachable and reports a parseable version). do we still want to do that just to check the error, e.g.

```go
_, err := getUpgradeContext(f.ClientSet.Discovery(), *upgradeTarget)
framework.ExpectNoError(err)
```

Reply:
actually, the passed-in upgCtx is not always constructed using this same method, e.g.:

https://github.com/kubernetes/kubernetes/blob/3edbc6afff17ea8dfe5c10b2677dcdc8767f67e2/test/e2e/lifecycle/cluster_upgrade.go#L196